app/services/idv/proofing_components.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Idv
+  class ProofingComponents
+    def initialize(
+        user:,
+        idv_session:
+      )
+      @user = user
+      @idv_session = idv_session
+    end
+
+    def document_check
+      if user.establishing_in_person_enrollment || user.pending_in_person_enrollment
+        Idp::Constants::Vendors::USPS
+      elsif idv_session.remote_document_capture_complete?
+        DocAuthRouter.doc_auth_vendor(
+          discriminator: idv_session.document_capture_session_uuid,
+        )
+      end
+    end
+
+    def document_type
+      return 'state_id' if idv_session.remote_document_capture_complete?
+    end
+
+    def source_check
+      Idp::Constants::Vendors::AAMVA if idv_session.verify_info_step_complete?
+    end
+
+    def resolution_check
+      Idp::Constants::Vendors::LEXIS_NEXIS if idv_session.verify_info_step_complete?
+    end
+
+    def address_check
+      if idv_session.verify_by_mail?
+        'gpo_letter'
+      elsif idv_session.address_verification_mechanism == 'phone'
+        'lexis_nexis_address'
+      end
+    end
+
+    def liveness_check
+      # Not used
+    end
+
+    def device_fingerprinting_vendor
+      # Not used
+    end
+
+    def threatmetrix
```

Comment:
As it stands, this flag seems to track "was `proofing_device_profiling_collecting_enabled` on when the user went through identity resolution?"